packages/types/src/Moment.ts

```diff
@@ -27,8 +27,6 @@ export default class Moment extends Base<Date> {
       return value.raw;
     } else if (value instanceof Date) {
       return new Date(Math.ceil(value.getTime() / 1000) * 1000);
-    } else if (value instanceof UInt) {
```

Comment:
In removing this, we don't seem to cater for `BN` construction anymore? Should we not have a `isBn` path as well?

Reply:
The return at the bottom of the function handles the BN case.